internal/v2/controller/http/controller.go

```diff
@@ -111,29 +111,29 @@ func (v2c *V2HttpController) Metrics(writer http.ResponseWriter, request *http.R
 	v2c.sendResponse(writer, request, contractsV2.ApiMetricsRoute, response, http.StatusOK)
 }
 
-// Secrets handles the request to add App Service exclusive secrets to the Secret Store
+// AddSecret handles the request to add App Service exclusive secret to the Secret Store
 // It returns a response as specified by the V2 API swagger in openapi/v2
-func (v2c *V2HttpController) Secrets(writer http.ResponseWriter, request *http.Request) {
+func (v2c *V2HttpController) AddSecret(writer http.ResponseWriter, request *http.Request) {
 	defer func() {
 		_ = request.Body.Close()
 	}()
 
-	secretRequest := common.SecretsRequest{}
+	secretRequest := common.SecretRequest{}
 	err := json.NewDecoder(request.Body).Decode(&secretRequest)
 	if err != nil {
 		v2c.sendError(writer, request, errors.KindContractInvalid, "JSON decode failed", err, "")
 		return
 	}
 
-	path, secrets := v2c.prepareSecrets(secretRequest)
+	path, secrets := v2c.prepareSecret(secretRequest)
```

Comment:
should the return variable also change to `secret`?

Reply:
Yes

Reply:
fixed